.editorconfig

```diff
@@ -0,0 +1,13 @@
+root = true
+
+[*]
+indent_style = space
+indent_size = 2
```

Comment:
We use 4 spaces for indentation.

Reply:
scroll down for the `.hs` section :)

Reply:
I think in practice this means it applies to the nix code.

Reply:
Ah okay.